scripts/configmap/create_cm.sh

```diff
@@ -1,12 +1,164 @@
-k delete cm powerscale-driver-v2.2.0,driver-sidecars,authorization-v2.2.0,authorization-common -n dell-csm-operator
-kubectl create cm powerscale-driver-v2.2.0 --from-file=powerscale/v2.2.0/ -n dell-csm-operator
-kubectl get cm powerscale-driver-v2.2.0 -n dell-csm-operator
+#!/bin/bash
 
-kubectl create cm driver-sidecars --from-file=sidecars/ -n dell-csm-operator
-k get cm driver-sidecars -n dell-csm-operator
+source ../common.bash
```

Comment:
why source this from top level folder  ?
no other script is using this common.sh --so its not really common ?

Reply:
I can't answer why no one else is using the script